cli_tools/gce_vm_image_export/main.go

```diff
@@ -50,6 +52,18 @@ func main() {
 		*network, *subnet, *zone, *timeout, *scratchBucketGcsPath, *oauth, *ce, *gcsLogsDisabled,
 		*cloudLogsDisabled, *stdoutLogsDisabled, *labels, currentExecutablePath); err != nil {
 
+		defer func() {
```

Comment:
Why is this deferred? Can't it be run directly here before log.Fatal?

Reply:
This func() is useless so far, so I used defer to avoid its execution. log.Fatal() will call to os.Exit(), which will skip deferred func.